pkg/model/context.go

```diff
@@ -178,6 +181,8 @@ func (b *KopsModelContext) CloudTagsForInstanceGroup(ig *kops.InstanceGroup) (ma
 		switch b.Cluster.GetCloudProvider() {
 		case kops.CloudProviderHetzner:
 			labels[hetzner.TagKubernetesNodeLabelPrefix+k] = v
+		case kops.CloudProviderGCE:
+			// TODO: Do nothing for now while we figure out how to address GCE label length limit of 63
```

Comment:
I'm guessing the label is too long for GCE, and the cluster-autoscaler doesn't look at this label anyway on GCE?